crates/bevy_sprite/src/mesh2d/mesh.rs

```diff
@@ -342,6 +349,7 @@ impl GetBatchData for Mesh2dPipeline {
     type Param = SRes<RenderMesh2dInstances>;
     type CompareData = (Material2dBindGroupId, AssetId<Mesh>);
     type BufferData = Mesh2dUniform;
+    type BufferInputData = ();
```

Comment:
This is likely causing the crash as casting slices of ZSTs to slices of u8 is likely not going to work. Glad this caught this at compile time and not at runtime :)

Reply:
Well, a `BufferVec<()>` would never actually be constructed. But the compiler still complained. Anyway, it's fixed now.